packages/cli/src/lib/strings.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+type Messages = {
+  promptHostMessage: string;
```

Comment:
Maybe we could split these up - eg have a different object for errors and prompts?

Or maybe not - I think it's fine as it is.

Reply:
Yeah, I'm not sure it's worth splitting. I'm just using this a convention for where the messages are used. My guess is that we may have the same string used in different ways.